### DISCUSSION TITLE
NVidia fan "0% override" not working

### DISCUSSION THREAD

**Discussion by jgoldshlag2** [Rem0o/FanControl.Releases#1193] [Q&A]
I eagerly updated Fan Control today thinking I'd finally be able to shut my nvidia 3060 Ti's fans off when it's cool, but alas, the ~30% minimum fan speed is still in effect.

How exactly does this new feature work? I don't see anything about it in the program itself. I've tried a flat curve at 0% but it doesn't work, it just lowers to 30% and sits there with the little questions mark before the 30%.

**Comment by Rem0o** [comment]
![image](https://user-images.githubusercontent.com/14118956/196832352-44363a94-4219-445d-87cf-f3a72d7f9633.png)

This won't set your card to 0 RPM. It will basically disable Fan Control for that specific control when the % is set to 0 from the software ( manually set to 0 should do that part). If without Fan Control your card hits 0, this should work, otherwise, it won't do anything.

**Reply by jgoldshlag2** [reply]
So since it looks like the BIOS for my card (MSI Ventus 3060 Ti) doesn't support 0rpm for the fans, I'm out of luck?

That's a bummer.

My previous card (a 2060 Super FE) I re-flashed to an EVGA BIOS that supported 0rpm on the fans and it worked fine.

**Reply by duckduckdog** [reply]
I can't promise it's still possible, but I used NVflash to read my bios & a tool  (I think nibitor) to edit the minimum fan speed to zero before flashing the modified bios to the card.  I never felt comfortable flashing another card's BIOS.
